.github/workflows/test.yml

```diff
@@ -561,5 +559,185 @@ jobs:
       - uses: codecov/codecov-action@v2
         with:
           files: ./packages/generator-helper/src/__tests__/coverage/clover.xml
-          flags: generator-helper,${{ matrix.os }}
+          flags: generator-helper,${{ matrix.os }},library,binary
+          name: generator-helper-${{ matrix.os }}
+
+  #
+  # Run all tests on macOS and Windows.
+  #
+  # Unlike the other jobs, this job doesn't use Docker (and thus skips some
+  # tests that require dependencies not easily installable without Docker —
+  # specifically, SQL Server), and runs tests on Node.js 12, 14 and 16 (Linux
+  # jobs only run on Node.js 12 and 16 because Node.js 14 is tested in
+  # Buildkite).
+  #
+  # It also runs the tests for different packages sequentially to prevent
+  # combinatorial explosion of the number of parallel jobs, and to minimize the
+  # number of times we need to install MySQL using the package manager
+  # (PostgreSQL and MongoDB are provided by GitHub Actions out of the box).
```

Comment:
Is this correct re Postgres? We seem to run `setup-postgres.sh` below.

Reply:
`setup-postgres.sh` only configures it, but doesn't install

Reply:
With MySQL and MongoDB we rely on the defaults to keep things simple, but with Postgres it's a little bit more complicated, because there are no good/documented defaults on the macOS runner.